<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>508</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>172</red>
        <green>172</green>
        <blue>172</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>172</red>
        <green>172</green>
        <blue>172</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>172</red>
        <green>172</green>
        <blue>172</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>172</red>
        <green>172</green>
        <blue>172</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>173</red>
         <green>173</green>
         <blue>173</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>173</red>
         <green>173</green>
         <blue>173</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>173</red>
         <green>173</green>
         <blue>173</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>173</red>
         <green>173</green>
         <blue>173</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="1">
         <widget class="QPushButton" name="pbN0">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="shortcut">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pbN8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>8</string>
          </property>
          <property name="shortcut">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3" rowspan="2">
         <widget class="QPushButton" name="pbCalculate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>=</string>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pbRemove">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
          <property name="shortcut">
           <string>Backspace</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pbN1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
          <property name="shortcut">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pbN4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>4</string>
          </property>
          <property name="shortcut">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pbN2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2</string>
          </property>
          <property name="shortcut">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pbN6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>6</string>
          </property>
          <property name="shortcut">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pbN3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>3</string>
          </property>
          <property name="shortcut">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pbN9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>9</string>
          </property>
          <property name="shortcut">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pbN7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>7</string>
          </property>
          <property name="shortcut">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="pbDot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>.</string>
          </property>
          <property name="shortcut">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pbN5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5</string>
          </property>
          <property name="shortcut">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pbClear">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pbAdd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>+</string>
          </property>
          <property name="shortcut">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pbSubtract">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="shortcut">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pbMultiply">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>*</string>
          </property>
          <property name="shortcut">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pbDivide">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>/</string>
          </property>
          <property name="shortcut">
           <string>/</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="leResult">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>32</number>
        </property>
        <item>
         <widget class="QLineEdit" name="leLeftValue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leOperator">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leRightValue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbN0</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputNum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbN1</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputNum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbN2</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputNum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbN3</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputNum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbN4</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputNum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbN5</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputNum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbN6</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputNum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbN7</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputNum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbN8</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputNum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbN9</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputNum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAdd</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputOperator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCalculate</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputOperator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClear</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputOperator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbDivide</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputOperator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbDot</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputOperator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbMultiply</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputOperator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbRemove</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputOperator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSubtract</sender>
   <signal>clicked()</signal>
   <receiver>Calculator</receiver>
   <slot>inputOperator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputNum()</slot>
  <slot>inputOperator()</slot>
 </slots>
</ui>
